configure

```diff
@@ -5795,6 +5801,8 @@ QBT_ADD_DEFINES="$QBT_ADD_DEFINES $QBT_CONF_DEFINES"
 
 
 
+
+
```

Comment:
@Chocobo1:  Did you mean to add these two blank lines? 

They do not seem to be related to any other changes in this file.

Reply:
"configure" is auto generated by Autotools.